Gemfile

```diff
@@ -56,3 +56,6 @@ end
 
 gem 'pdfcrowd'
 gem 'google-cloud-translate'
+
+gem 'omniauth-auth0', '~> 3.0'
+gem 'omniauth-rails_csrf_protection', '~> 1.0' # prevents forged authentication requests
```

Comment:
Do we need this for SPAs? I'm not actually even sure how we can make this work with our app, since we would need a way to extract the CSRF token from the API responses in our client-side app.

Reply:
This was part of the example code, i can try it without and see if it works.